code/backend/batch/utilities/helpers/embedders/IntegratedVectorizationEmbedder.py

```diff
@@ -0,0 +1,40 @@
+from .EmbedderBase import EmbedderBase
+from ..EnvHelper import EnvHelper
+from ..LLMHelper import LLMHelper
+from ...integrated_vectorization.AzureSearchIndex import AzureSearchIndex
+from ...integrated_vectorization.AzureSearchIndexer import AzureSearchIndexer
+from ...integrated_vectorization.AzureSearchDatasource import AzureSearchDatasource
+from ...integrated_vectorization.AzureSearchSkillset import AzureSearchSkillset
+from ..config.ConfigHelper import ConfigHelper
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class IntegratedVectorizationEmbedder(EmbedderBase):
+    def __init__(self, env_helper: EnvHelper):
+        self.env_helper = env_helper
+        self.llm_helper: LLMHelper = LLMHelper()
+        pass
+
+    def embed_file(self, source_url: str, file_name: str):
+        self.process_using_integrated_vectorization(source_url=source_url)
```

Comment:
Is there a reason why there's a separate function for `process_using_integrated_vectorization`? Could it not all happen in `embed_file`?

Reply:
`embed_file` is the abstract method for PushEmbedder & IVEmbedder. `embed_file`in PushEmbedder has specific functionality, so to maintain the same pattern for both Push & IV, created a separate method